Fika Fika A13 + Photos

matthiasrossini opened the pull request:
Excuse my bad coding skills, I think this time the .DS_Store should be removed and I tested the page on my localhost, so should be fully functional :)

pqvst commented:
In case you haven't already, be sure to run `yarn run resize` or `npm run resize` to compress the images :)

I should update the readme... 😅

matthiasrossini commented:
ok wait let me do that first and then I'll re-submit, did not consider that factor

matthiasrossini commented:
Ok, should be mergeable now. Let me know if this works. 
- added fika fika a13 + description + images
- added "images insert" + "resizing" to the readme
- added .DS_Store to the gitignore file
